h/accounts/models.py

```diff
@@ -78,6 +78,9 @@ class User(Base):
                       default=False,
                       server_default=sa.sql.expression.false())
 
+    sidebar_tutorial_dismissed = sa.Column(
+        sa.Boolean, default=False, server_default=sa.sql.expression.false())
```

Comment:
Done. Fwiw I originally fell back to this style rather than introduce a `\` or `(...)`'s because if I follow the style used above this line then `server_default=sa.sql.expression.false()` doesn't fit on one line.